apps/nextjs-website/src/components/molecules/WebinarQuestion/WebinarQuestionRow.tsx

```diff
@@ -21,104 +29,102 @@ export default function WebinarQuestionRow({
-              : palette.action.hover,
-        },
-        backgroundColor:
-          isHighlighted && !isHidden ? palette.primary.light : '',
-        fontStyle: isHidden ? 'italic' : '',
-        position: 'relative',
-      }}
-    >
-      <TableCell
-        sx={{
-          color: tcColor,
-        }}
-      >
-        {!isHidden
-          ? formatter.dateTime(question.id.createdAt, {
-              year: 'numeric',
-              month: 'numeric',
-              day: 'numeric',
-              hour: 'numeric',
-              minute: 'numeric',
-            })
-          : ''}
-      </TableCell>
-      <TableCell
-        width='70%'
+    (!isHidden || isHiddenByMe) && (
+      <TableRow
+        hover
+        key={question.id.createdAt.toISOString()}
         sx={{
-          color: tcColor,
+          '&:last-child td, &:last-child th': { border: 0 },
+          '&.MuiTableRow-hover:hover': {
+            backgroundColor:
+              isHighlighted && !isHidden
+                ? palette.primary.dark
+                : palette.action.hover,
+          },
+          '& > .MuiTableCell-root': {
+            padding: '32px 16px 24px 16px',
+          },
+          backgroundColor:
+            isHighlighted && !isHidden ? palette.primary.light : '',
+          fontStyle: isHidden ? 'italic' : '',
+          position: 'relative',
         }}
       >
-        {!isHidden
-          ? question.question
-          : hiddenBy === userName
-          ? `${t('hiddenByMe')}: (${question.question})`
-          : `${t('hiddenBy')}: ${hiddenBy}`}
-      </TableCell>
-      <TableCell>
-        <Box display={'flex'} justifyContent={'space-between'}>
-          {(!isHidden || (isHidden && hiddenBy === userName)) && (
-            <IconButton
-              onClick={() => onHide(!isHidden)}
-              sx={{ color: tcColor }}
+        <TableCell
+          width='100%'
+          sx={{
+            color: tcColor,
+            '& > .MuiTypography-root': {
+              color: tcColor,
+            },
+          }}
+        >
+          {isHighlighted && !isHidden && (
+            <Box
+              sx={{
+                fontSize: 14,
+                color: palette.common.white,
+                marginBottom: 2,
+              }}
             >
-              {!isHidden ? <VisibilityOffIcon /> : <Visibility />}
-            </IconButton>
+              {t('highlightedBy')}: {highlightedBy}
+            </Box>
           )}
+          <Typography fontStyle={'italic'} fontSize={14}>
+            {!isHidden
+              ? question.id.createdAt.toLocaleTimeString(
+                  defaultLocale,
+                  timeOptions
+                )
+              : ''}
+          </Typography>
+          <Typography>
+            {!isHidden
+              ? question.question
+              : isHiddenByMe
+              ? `${t('hiddenByMe')}: (${question.question})`
+              : `${t('hiddenBy')}: ${hiddenBy}`}
```

Comment:
Now we can't see hidden question from other user anymore. So i think we can simplfy this in:
```suggestion
            {isHiddenByMe
              ?  `${t('hiddenByMe')}: (${question.question})`
              : question.question}
```